stdlib/camlinternalFormat.ml

```diff
@@ -626,7 +626,6 @@ let bprint_fmt buf fmt =
       bprint_string_literal buf (string_of_formatting_lit fmting_lit);
       fmtiter rest ign_flag;
     | Formatting_gen (fmting_gen, rest) ->
-      bprint_string_literal buf "@{";
```

Comment:
I think that the use of `bprint_string_literal` below is also a bug: it renders `@[<%s>@]` as `@[<%%s>@]`.